types.d.ts

```diff
@@ -1,79 +1,187 @@
+// tslint:disable:interface-name

```

Comment:
There is no TSLint in this project so we don't need an instruction for it except there is a case where it breaks code for people using TSLint and our library. If `// tslint:disable:interface-name` gets removed, will TSLint complain when "binance-api-node" is used as a dependency?